index.d.ts

```diff
@@ -0,0 +1,52 @@
+declare module '@articulate/authentic' {
+  interface VerifyOpts {
+    algorithms?: string[]
+    audience?: string | RegExp | Array<string | RegExp>
+    clockTimestamp?: number
+    clockTolerance?: number
+    issuer?: string | string[]
+    ignoreExpiration?: boolean
+    ignoreNotBefore?: boolean
+    jwtid?: string
+    subject?: string
+    maxAge?: string
+  }
+
+  interface JWKSOpts {
+    cache?: boolean,
+    rateLimit?: boolean
+  }
+
+  interface AuthenticOpts {
+    issWhitelist: string[],
+    jwks?: JWKSOpts,
+    verify?: VerifyOpts
+  }
+
+  interface Validator {
+    (token: string): Promise<Authentic.JWT>
+  }
+
+  function Authentic(opts: AuthenticOpts): Validator
+
+  namespace Authentic {
+    interface JWT {
+      // Standard JWT claims
+      aud: string,
+      cid?: number,
+      exp: number,
+      iat: number,
+      iss: string
+      jti?: number,
+      scp?: Array<String>
+      sub: string
+      uid?: number,
+
+      // Custom JWT claims
+      aid?: string,
```

Comment:
@vscerchia Now that I've moved this into `@articulate/authentic`, having our custom JWT claims here seems wrong. Should we extend the JWT interface in our code and add our custom claims there?

Also, what are your thoughts on adding `[key: string]: any` to the JWT interface? I think we should so anyone using this package can add their custom claims here.

Reply:
Hey @tylerodonnell, I agree we should remove the custom claims and add `[key: string]: any` here.

Reply:
👍 Done in 895bacf